clients/src/main/java/org/apache/kafka/clients/consumer/internals/PartitionAssignor.java

```diff
@@ -202,11 +202,11 @@ public String toString() {
 
     class Assignment {
         private final Short version;
-        private final List<TopicPartition> partitions;
+        private List<TopicPartition> partitions;
         private final ByteBuffer userData;
-        private final ConsumerProtocol.Errors error;
+        private ConsumerProtocol.AssignmentError error;
```

Comment:
Does the assignor need to be aware of this error? If it is just for internal use, can we leave it out of the public API?

Reply:
Yup, ack.